### DISCUSSION TITLE
Closed parent appearing in active task list when open children are not yet active

### DISCUSSION THREAD

**Discussion by reagle** [zim-desktop-wiki/zim-desktop-wiki#1914] [Ideas]
[Porting from the mailing list on @jaap-karssenberg's recommendation.] 

"Upper level" tasks -- even if closed -- appear in the Task List as active even if their sub-items are not *yet* active (because of the date) and not shown themselves.

That is, shouldn't the behavior be: if an open checkbox that isn't yet active because of its date is precluded from the active item list, so should it's closed parent. And then, once a child is within the active date range, it and its parent reappear in the task list.

```
[*] @misc
    [ ] >2022-04 misc item 1 <2022-06
    [ ] >2022-04 item 2 <2022-04-21
    [ ] >2022-04 item 3
    [ ] >2022-05 item 4

**Comment by jaap-karssenberg** [comment]
I've checked the behavior with the new tasklist version. There the parent
is only shown when the children are also shown in a hierarchic list (greyed
out because they are not started). In the "active" list both the parent and
the children are hidden.

Regards,

Jaap

**Comment by reagle** [comment]
I just tried the latest and it isn't finding *any* of my tasks. `show open tasks` is selected as is the property `Consider all checkboxes as tasks` (I don't use labels much).

Also, most confusing, when I click `help` in the task plugin property view, a new zim window is opened with the documentation wiki but I can't interact with it or the original wiki until the preferences dialog is dismissed.

**Comment by jaap-karssenberg** [comment]
Probably you're index rebuild did not complete when changing versions.
Might need to trigger it manually.

The help window is indeed annoying &  but already captured in the issue
tracker.

**Comment by reagle** [comment]
I did manually trigger the command to rebuild the index, but that wasn't sufficient. Deleting the cache folder is more effective: `rm Library/Caches/org.zim-wiki.Zim/`.

Even so, task items are not correctly showed as nested/inheriting items in my existing notebook. When I created a new notebook, they seemed to be working correctly.
